File: DOTNET/BaiduMapTest/PlaceDetailTest.cs
using BaiduMap.Request;
using BaiduMap.Request.Models;
using BaiduMap.Response;
using System;
using System.Collections.Generic;
using System.Text;
using Xunit;
using Shouldly;

namespace BaiduMapTest
{
    public class PlaceDetailTest
    {
        [Fact]
        public void Test()
        {
            var baiduClient = ClientCreator.Create();
            var detailMode = new PlaceDetailModel()
            {
                Uid = "d5101afd619d12c7f423100e",
            };
            
            var req = new PlaceDetailRequest(detailMode);
            PlaceDetailResponse resp = baiduClient.Execute(req);
            resp.Result.ShouldNotBeNull();
        }
    }
}

File: JAVA/baidumap-webapi/src/test/java/wang/seamas/baidumap/PlaceDetailTest.java
package wang.seamas.baidumap;

import org.junit.Assert;
import org.junit.Test;
import wang.seamas.baidumap.request.PlaceDetailRequest;
import wang.seamas.baidumap.request.model.PlaceDetailModel;
import wang.seamas.baidumap.response.PlaceDetailResponse;

/**
 * @author: Seamas Wang
 * @description:
 * @date: Created in 11:56 2018/8/6
 */
public class PlaceDetailTest {

    @Test
    public void test()
    {
        BaiduApiClient baiduClient = ClientCreator.Create();
        PlaceDetailModel model = new PlaceDetailModel();
        model.setUid("d5101afd619d12c7f423100e");

        PlaceDetailRequest request = new PlaceDetailRequest(model);
        PlaceDetailResponse response = baiduClient.execute(request);
        Assert.assertEquals(response.getStatus(), 0);
    }
}
